src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.cs

```diff
@@ -994,12 +1051,18 @@ public bool StatusCanHaveBody(int statusCode)
                    statusCode != 304;
         }
 
-        public void ReportCorruptedHttpRequest(BadHttpRequestException ex)
+        public void ReportBadHttpRequest(BadHttpRequestException ex)
```

Comment:
For that matter, why bother constructing the exception if it's never thrown, only logged? Just log the message directly.